src/main/java/uk/gov/hmcts/sptribs/notification/NotificationHelper.java

```diff
@@ -182,14 +182,14 @@ public void setRecordingTemplateVars(Map<String, Object> templateVars, Listing l
     }
 
     public Map<String, String> buildDocumentList(DynamicMultiSelectList documentList, int docAttachLimit) {
-        Map<String, String> uploadedDocuments = new HashMap<>();
+        final Map<String, String> uploadedDocuments = new HashMap<>();
 
         int count = 0;
         if (!ObjectUtils.isEmpty(documentList.getValue()) && documentList.getValue().size() > 0) {
```

Comment:
You can replace this condition with `ObjectUtils.isNotEmpty(documentList.getValue())`